src/pages/index.js

```diff
@@ -90,28 +88,6 @@ const Main = styled.div`
   color: #c6e2ff;
 `;
 
-const LeaderboardLink = styled.div`
```

Comment:
Were these redundant?

Reply:
Not redundant but unused. We don't have leaderboard so need of style for leaderboard.
Btw what happened to leaderboard anyway?

Reply:
@Yugandhartripathi it was causing build issues. I will upload the leaderboard again and see if it works with the updated dependencies.

Reply:
There was an error with octokit when we tried to build the project. Me and @ryzokuken tried solving the bug but were not successful. Let's try it again this time and make leaderboard great again.

Reply:
Don't worry about the design. Everything is done, we just need to figure out the issue with ocktokit.

Reply:
Not about the design of the leaderboard page but where are we gonna add a link to it in the main page? We are planning to add all these links to our page like one for talk proposals and leaderboard and then maybe a place for displaying confirmed talks with time venue details. So if it's design is decided already then I would like to know what was decided plus when because I would like to be part of the next such discussion session.